cordapp-upgrades/build.gradle

```diff
@@ -0,0 +1,147 @@
+apply plugin: 'kotlin'
+apply plugin: 'net.corda.plugins.cordapp'
+apply plugin: 'net.corda.plugins.cordformation'
+
+buildscript {
+    ext {
+        corda_release_group = 'net.corda'
+        corda_release_version = '5.0-20190401.000019-81'
```

Comment:
Add a TODO here on changing this to 4.1 when it's released. Can you also create a JIRA ticket to track this.